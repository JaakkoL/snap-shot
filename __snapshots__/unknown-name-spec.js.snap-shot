exports['11558f62573f97cac76804be34258b1880295ee27ff38f280bb6b4aeaf94f456 1'] = 30

exports['handles template literals ${secondTest} 1'] = 30

exports['testThree 1'] = "foo"

exports['testThree 2'] = "bar"

exports['7464a8f45830ac78b6589a86dfacd4f080089054aee432fd0e339e5f691e94a8 1'] = "test A works"

exports['7464a8f45830ac78b6589a86dfacd4f080089054aee432fd0e339e5f691e94a8 2'] = "test B works"

exports['7464a8f45830ac78b6589a86dfacd4f080089054aee432fd0e339e5f691e94a8 3'] = "test C works"

exports['testSomething 1'] = "test A works"

exports['testSomething 2'] = "test B works"

exports['testSomething 3'] = "test C works"

