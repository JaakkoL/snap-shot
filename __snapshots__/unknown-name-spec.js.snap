module.exports[`11558f62573f97cac76804be34258b1880295ee27ff38f280bb6b4aeaf94f456 0`] = 30

module.exports[`handles template literals ${secondTest} 0`] = 30

